lib/ansible/module_utils/facts/system/date_time.py

```diff
@@ -18,9 +18,10 @@
 from __future__ import (absolute_import, division, print_function)
 __metaclass__ = type
 
-import datetime
 import time
 
+from datetime import datetime
```

Comment:
I'd be cautious doing this. Because the class is the same name as the module, this often trips people up.  I know it's  longer, but I just rely on using `datetime.datetime.` everywhere, to avoid the potential confusion that this import can create.

Reply:
That's fine. I think this also made the patching a bit cleaner because I could patch the `datetime` imported in `ansible.module_utils.facts.system.date_time`, but I can change it.